src/pages/product-list/components/display/display.tsx

```diff
@@ -26,7 +26,15 @@ interface IProps {
   onInit: () => void;
   onOptionsChange: (options: PageOptions) => void;
 }
-
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '&:hover': {
+        textDecoration: 'none'
+      }
```

Comment:
We should avoid adding styles inside so we can have a clean component. That's why we use the styled components. This can be achieved with that

Reply:
I couldn't fetch the correct css class with StyledComponents ...